### ISSUE TITLE
compatibility with Laravel 7.0?

### ISSUE THREAD

**Issue by obaid** [centrality-labs/spark-tailwindcss#1]
Any idea if this works with Laravel 7.0? If so can we bump it in composer.json?

**Comment by iantasker** [comment]
Hi @obaid 

I haven't tried it with laravel 7 yet, if you need me to bump it urgently then please submit a pr otherwise i will test compatibility over the weekend.

**Comment by fwartner** [comment]
It doesn't...

**Comment by iantasker** [comment]
@fwartner Do you have any information on where it's failing?

**Comment by fwartner** [comment]
Since Laravel 7 has changed the way how presets will be processed, you´ll need to use the UI implementation. Also there´s a problem with the stubs which wont be replaced correctly.

**Comment by iantasker** [comment]
@fwartner Thanks, I will investigate further.

**Comment by fwartner** [comment]
Awesome! :)

**Comment by iantasker** [comment]
@fwartner Before I get into this have you come across any compatibility issues between Laravel Spark 9 and Laravel 7?

**Comment by fwartner** [comment]
Can’t say.

Von meinem iPhone gesendet

**Comment by iantasker** [comment]
I have created a PR #3 which adds support for Laravel 7.0, it has been released as a `2.0.0-alpha` package. 

@obaid / @fwartner Please could you let me know if you encounter any issues?

**Comment by sandervanhooft** [comment]
@iantasker I have tried Spark 9 with Laravel 7 and can confirm the default setup works, as long as you pull in `laravel/ui`.

**Comment by shaunthornburgh** [comment]
I'm trying to install with Laravel 7 and am getting the following:

$ composer require centrality-labs/spark-tailwindcss:2.0.0 --dev

  [InvalidArgumentException]                                                            
  Could not find package centrality-labs/spark-tailwindcss in a version matching 2.0.0

**Comment by shaunthornburgh** [comment]
I am running Laravel 7, I was able to install this using `composer require centrality-labs/spark-tailwindcss:2.0.0-alpha --dev`

However when I run `php artisan ui spark-tailwindcss` I get the following error

```
Symfony\Component\ErrorHandler\Error\FatalError 

  Cannot use Laravel\Ui\Presets\Preset as Preset because the name is already in use

  at vendor/centrality-labs/spark-tailwindcss/src/SparkTailwindCssPreset.php:9
     5| use Illuminate\Support\Arr;
     6| use Illuminate\Container\Container;
     7| use Illuminate\Filesystem\Filesystem;
     8| use Illuminate\Foundation\Console\Presets\Preset;
  >  9| use Laravel\Ui\Presets\Preset;
    10| 
    11| class SparkTailwindCssPreset extends Preset
    12| {
    13|     public static function install()


   Whoops\Exception\ErrorException 
```

**Comment by iantasker** [comment]
@shaunthornburgh I have now merged PR #3 which included a fix for this.
